src/univ/menu.cpp

```diff
@@ -180,10 +180,11 @@ class wxPopupMenuWindow : public wxPopupTransientWindow
     virtual void DoDraw(wxControlRenderer *renderer);
 
     // event handlers
-    void OnLeftUp(wxMouseEvent& event);
+    void OnLeftDown(wxMouseEvent& event);
```

Comment:
Not sure why has been "down" changed to "up"? At least under MSW the menu items are definitely activated when the mouse is released ("up"), not pressed ("down"), so this doesn't seem right to me.

Reply:
I did so temporarily to simplify testing because there was mess between left_up that was handled only in menu and left_down that was handled only in menubar.